plasmapy/simulation/tests/test_particletracker.py

```diff
@@ -132,75 +140,83 @@ def test_particle_exb_drift(uniform_magnetic_field):
                           atol=1e-3 * u.m / u.s), \
             "x velocity doesn't agree with expected drift velocity!"
 
-    s.test_kinetic_energy()
-
     # s.plot_trajectories()
+    s.test_kinetic_energy()
 
 
-""" TODO: figure out how to get this test to work
+@pytest.mark.skip
```

Comment:
I'll be trying to do something interesting with this one, but I might just use a more interesting field and stick that into the example.